Tests/SwiftProtobufTests/unittest.pb.swift

```diff
@@ -6947,6 +6991,18 @@ struct ProtobufUnittest_TestRequiredOneof: SwiftProtobuf.Message, SwiftProtobuf.
 
     init() {}
 
+    var isInitialized: Bool {
```

Comment:
Does it make sense to try to optimize this in the generator, and if only one case of the oneof is a message that needs to be checked, instead generate an if to test that one instead of a switch?